src/core/components/model-example.jsx

```diff
@@ -8,18 +8,23 @@ export default class ModelExample extends React.Component {
     specSelectors: PropTypes.object.isRequired,
     schema: PropTypes.object.isRequired,
     example: PropTypes.any.isRequired,
+    examples: ImPropTypes.orderedMap,
     isExecute: PropTypes.bool,
     getConfigs: PropTypes.func.isRequired,
     specPath: ImPropTypes.list.isRequired,
   }
 
   constructor(props, context) {
     super(props, context)
-    let { getConfigs } = this.props
+
+    let { getConfigs, examples } = this.props
     let { defaultModelRendering } = getConfigs()
     if (defaultModelRendering !== "example" && defaultModelRendering !== "model") {
       defaultModelRendering = "example"
     }
+    if (defaultModelRendering === "example" && examples) {
+      defaultModelRendering = `example_${ examples.keySeq().first() }`
+    }
     this.state = {
       activeTab: defaultModelRendering
```

Comment:
The `activeTab` value should also be updated when switching between content types in case they provide different examples. Currently it fails to load a tab if the examples don't match.